external-import/cve/src/cve.py

```diff
@@ -68,7 +68,7 @@ def convert_and_send(self, url, work_id):
                 file.write(image)
             # Unzipping the file
             self.helper.log_info("Unzipping the file")
-            with gzip.open("data.json.gz", "rb") as f_in:
+            with gzip.open(os.path.dirname(os.path.abspath(__file__)) + "data.json.gz", "rb") as f_in:
```

Comment:
Hey @izm1chael 

Thanks for your PR, please fix the missing slash /

Regards

```suggestion
            with gzip.open(os.path.dirname(os.path.abspath(__file__)) + "/data.json.gz", "rb") as f_in:
```